Fixtures/BinaryTargets/Framework.xcframework/macos-x86_64/Framework.framework/Versions/A/Headers/Framework.h

```diff
@@ -0,0 +1,19 @@
+//
+//  Framework.h
+//  Framework
+//
+//  Created by David Hart on 14.01.20.
+//  Copyright © 2020 David Hart. All rights reserved.
```

Comment:
Can you change this to Swift.org copyright headers instead?

Reply:
Fixed in future commit.